tests/testthat/setup.R

```diff
@@ -0,0 +1,101 @@
+# Collection of functions and data pre-processing to help with testing
+library(officer)
+library(xml2)
```

Comment:
I had to put them back here because they are called anyway outside test_that and it is better not to clutter too much with `::` either